src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from "classnames";
+import * as AvatarPrimitive from "@radix-ui/react-avatar";
+
+export type AvatarProps = {
+  src?: string;
+  text?: string;
+  size?: "medium" | "large";
+  className?: string;
+} & AlertDialog.DialogProps;
```

Comment:
Does `AlertDialog` need to be imported? If not, I'm probably just not seeing what is putting the namespace on the global scope.